libs/shared/ui/dock/dock.tsx

```
@@ -0,0 +1,170 @@
+        ? iconSize
+        : isSmallScreen
+          ? SMALL_SCREEN_SIZE
+          : DEFAULT_SIZE
+    const effectiveIconMagnification
+      = iconMagnification !== undefined
+        ? iconMagnification
+        : isSmallScreen
+          ? SMALL_SCREEN_MAGNIFICATION
+          : DEFAULT_MAGNIFICATION
+
+    const mouseX = useMotionValue(Infinity)
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)